schema/spacewalk/upgrade/susemanager-schema-4.2.4-to-susemanager-schema-4.2.5/900-actionchaindispatch.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE rhnActionChain DROP CONSTRAINT IF EXISTS rhn_actionchain_dispatched_ck;
```

Comment:
Head got tagged today. So new schema migration dir is 4.2.5-to-4.2.6

Reply:
JFYI - we just had to make another schema release. So just lookup the correct directory when the branch is open again to merge :-)